packages/flame_forge2d/example/lib/joints/constant_volume_joint.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:math';
+
+import 'package:flame/input.dart';
+import 'package:flame_forge2d/flame_forge2d.dart';
+import 'package:flame_forge2d_example/utils/ball.dart';
+import 'package:flame_forge2d_example/utils/walls.dart';
+
+class ConstantVolumeJointExample extends Forge2DGame with TapDetector {
+  @override
+  Future<void> onLoad() async {
+    super.onLoad();
+    addAll(createBoundaries(this));
+  }
+
+  @override
+  Future<void> onTapDown(TapDownInfo details) async {
+    super.onTapDown(details);
+    final tap = details.eventPosition.game;
```

Comment:
You could name this center directly so that you don't have to create the center variable further down